docs/collecting-data/collecting-from-own-applications/python-tracker/tracking-specific-events/index.md

```diff
@@ -1,528 +1,282 @@
-
-**_\*\*Prior to v0.9.0_**
+Every tracked event payload has a unique event_id UUID string (`eid`). Other properties include the name_tracker (`namespace`) and `app_id` set when the Tracker was initialized. From version 1 onwards, `tracker.track()` returns the payload's `eid`.
 
-Before version 0.9.0 of the Python Tracker, providing a `snowplow_tracker.timestamp.TrueTimestamp` object as the timestamp argument will attach a true timestamp to the event, replacing the device timestamp. For example:
+Snowplow events have a defined structure and protocol that is identical regardless of the tracker used. A minimal payload - the raw event - is sent from the tracker to your collector. The raw event is [enriched](/enriching-your-data/what-is-enrichment/index.md) as it passes through your pipeline. By the time the event arrives in your data storage, depending which [enrichments](/enriching-your-data/available-enrichments/index.md) you have enabled, it will have gained different kinds of metadata, and have many more fields than it started with.
 
-```python
-from snowplow_tracker.tracker import TrueTimestamp
-t.track_struct_event("some cat", "save action", tstamp=TrueTimestamp(1368725287000))
-```
-
-Above will attach [`ttm`](/docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md#timestamp-parameters)([`true_tstamp`](/docs/understanding-your-pipeline/canonical-event/index.md#date--time-fields)) parameter instead of default `dtm`. You can also use, plain integer, `DeviceTimestamp` or `None` to send `device_sent_timestamp`.
-
-**_\*\*New to v0.9.0_**
-
-Since version 0.9.0, providing the optional timestamp argument will only set the true timestamp (true_tstamp) of the event. The type of this argument can only be the unix time in milliseconds. If you migrate from previous version, make sure to replace any references to Timestamp objects, since the Timestamp class (along with the TrueTimestamp and DeviceTimestamp subclasses) do not exist.
-
-```python
+The Python tracker [Github repository](https://github.com/snowplow/snowplow-python-tracker) includes 3 example apps demonstrating different ways to send events to your collector.
 
-t.track_struct_event("some cat", "save action", tstamp=1368725287000)
-```
-
-#### 3. Event subject (since v0.9.0)
-
-Since version 0.9.0, it is possible to set the Subject per-event, in order to augment the event with extra information without having to change the Subject at the Tracker level. This provides a thread safe way to track multiple subjects.
-
-This is supported as an optional keyword argument by all track methods. For example:
+## Out-of-the-box Events
+The Python tracker provides classes for tracking different types of events. They are listed below:
 
-```python
-
-evSubject = Subject().set_user_id("1234")
-t.track_page_view("www.example.com", event_subject=evSubject)
-```
+| **Event** | **Description** |
+| --- | --- |
+| `SelfDescribing` | Track custom events with custom schemas |
```

Comment:
I'd argue that SelfDescribing are not really out of the box events, they aren't classed as such in other areas on the docs. I would maybe split this one out and add the section about it beneath all the others as it's a more complex use case.